phenol-cli/src/main/java/org/monarchinitiative/phenol/cli/demo/PairwisePhenotypicSimilarityCalculator.java

```diff
@@ -140,19 +138,19 @@ public PairwisePhenotypicSimilarityCalculator(String hpoPath,
     }
     boolean badFile=false;
     // check existence of Files
-    if (! (new File (this.pathHpObo).exists())) {
+    if (!pathHpObo.toFile().isFile()) {
       System.err.println("[ERROR] hp.obo file not found at "+pathHpObo);
       badFile=true;
     }
```

Comment:
Isn't the intention this:
```java
if (!Files.exists(pathHpObo)) {
    System.err.println("[ERROR] hp.obo file not found at "+pathHpObo);
}
```
or use `Files.isRegularFile(pathHpObo)`

Reply:
Yeah, `Files.isRegularFile` is the method I've been trying to mimic. Thanks!